chia/rpc/wallet_rpc_api.py

```diff
@@ -544,7 +544,7 @@ async def send_transaction(self, request):
         wallet_id = int(request["wallet_id"])
         wallet = self.service.wallet_state_manager.wallets[wallet_id]
 
-        if not isinstance(request["amount"], int) or not isinstance(request["amount"], int):
+        if not isinstance(request["amount"], int) or not isinstance(request["fee"], int):
```

Comment:
This looks good, but the code still looks fishy. 
Below we have a check `if "fee" in request:` but this would throw a key error at this point.
The 4 line if-clause below is also just the one line
```python
fee = uint64(request.get("fee", 0))
```
but I'm not sure how it will look like after the refactor.

Reply:
yeah I agree